app/modules/video-players/video-ads.js

```diff
@@ -201,14 +202,30 @@ function init(player, options, slotTargeting) {
 	});
 
 	player.on('adRequest', (event) => {
-		const vastParams = vastParser.parse(event.tag, {
-			imaAd: event.ima && event.ima.ad,
-		});
+		const timeStamp = Math.round(window.performance && window.performance.now && window.performance.timing &&
+			window.performance.timing.navigationStart ? window.performance.now() +
+			window.performance.timing.navigationStart : Date.now()),
+			vastParams = vastParser.parse(event.tag, {
+				imaAd: event.ima && event.ima.ad,
+			});
 		vastDebugger.setVastAttributes(videoContainer, 'success', vastParams);
 
 		if (options.featured) {
 			// featuredVideoDelay.markAsReady(vastParams.lineItemId);
 		}
+
+		slotTracker.onRenderEnded(
+			slot,
+			{
+				timestamp: timeStamp,
+				line_item_id: vastParams.lineItemId,
+				creative_id: vastParams.creativeId,
+				creative_size: vastParams.size,
+				status: 'success',
```

Comment:
We should track `error` too.
https://github.com/Wikia/app/blob/dev/extensions/wikia/AdEngine/js/video/player/jwplayer/jwplayerAdsTracking.js#L39

Reply:
Added.